File: sort/Java/sort.java
import java.util.ArrayList;
import java.util.Collections;

public class sort
{
    public static final int N = 1 << 22;
    public static void main(String[] args)
    {
        boolean print_out = false;
        if(args.length >= 1)
            print_out = true;
        ArrayList<Integer> v = new ArrayList<Integer>();
        int seed = 3;
        for(int i = 0; i < N; i++)
        {
            v.add(seed);
            seed *= 27487;
            seed %= 30491;
        }
        Collections.sort(v);
        if(print_out)
            System.out.println(v.get(279121));
    }
}

File: sort/C#/sort.cs
using System;
using System.Collections.Generic;

public class sort
{
    public const int N = 1 << 22;
    public static void Main(string[] args)
    {
        bool print_out = false;
        if(args.Length >= 1)
            print_out = true;
        List<int> v = new List<int>();
        int seed = 3;
        for(int i = 0; i < N; i++)
        {
            v.Add(seed);
            seed *= 27487;
            seed %= 30491;
        }
        v.Sort();
        if(print_out)
            Console.WriteLine(v[279121]);
    }
}
